cmake/std/sems/PullRequestGCC4.8.4TestingEnv.sh

```diff
@@ -12,9 +12,9 @@ source /projects/sems/modulefiles/utils/sems-modules-init.sh
 
 module load sems-gcc/4.8.4
 module load sems-openmpi/1.10.1
-module load sems-python/2.7.9
 module load sems-git/2.10.1
 module load sems-boost/1.63.0/base
+module swap sems-python/3.5.2
```

Comment:
If another python module isn't already loaded, will `swap` do anything?

Reply:
it will act as a load in that case.  Don't ask me why.